packages/discovery-react-components/tsconfig.prod.json

```diff
@@ -2,6 +2,17 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "declaration": true,
-    "isolatedModules": false
+    "isolatedModules": false,
+    "baseUrl": "src",
+    "paths": {
```

Comment:
can we put this in the `tsconfig.json` instead? or does it only work here?

Reply:
Yeah, I mentioned this in the PR description too, but if the `paths` are in the `tsconfig.json`, they get removed by `react-scripts` when running our tests. Hence placing them here and then also having to define them in our Storybook's tsconfig.